src/normalizeHTML.js

```diff
@@ -44,6 +44,8 @@ export default function normalizeHTML(html: string): string {
 
   // Convert every two consecutive "&nbsp;" into a spacer tab.
   html = html.replace(LONG_TAB_SPACE_PATTERN, TAB_SPACER_HTML);
+  // fix for jspdf copy paste text, from jspdf its always getting transparent
+  html = html.replace(/color:\s*transparent;/g, 'color: black;');
```

Comment:
This isn't the only way to set a color as transparent. Can you do something closer to "clear formatting" here? Or at least a bit more agnostic to what the color is actually set to? (for text and rgba values)

Reply:
This fix was for Paige team. They had a case: when a text is copied from pdf to licit, the text color was turning into transparent. I added this code to turn the transparent color to black.

Reply:
This doesn't address the original concern. This fix is extremely specific and fragile. If possible, it should be setup to clear all formatting as part of the paste operation.